lib/commonConfig.js

```diff
@@ -155,6 +155,7 @@ function processEnvironmentVariables() {
         'IOTA_MULTI_CORE',
         'IOTA_DEFAULT_EXPRESSION_LANGUAGE',
         'IOTA_RELAX_TEMPLATE_VALIDATION',
+        'IOTA_DEFAULT_ENTITY_NAME_CONJUNCTION',
```

Comment:
It should be included in documentation (env var and corresponding config.js parameter)

Reply:
fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/949/commits/832a13516e1e539d81cb52eafacfce51e3df8c0d